pyEpiabm/pyEpiabm/tests/test_unit/test_property/test_place_foi.py

```diff
@@ -67,15 +67,19 @@ def test_place_case_isolation(self):
                          result_isolating)
 
     def test_place_place_closure(self):
+        # Not place closure (closure_start_time = None)
         result = PlaceInfection.place_inf(self.place, self.infector, self.time)
         self.assertNotEqual(result, 0)
+
+        # Place closure
         self.infector.microcell.closure_start_time = 1
         result_closure = PlaceInfection.place_inf(self.place, self.infector,
                                                   self.time)
         self.assertEqual(result_closure, 0)
 
     def test_place_household_quarantine(self):
-        # Not in quarantine (quarantine_start_time = None)
+        # Update place type, not in quarantine (quarantine_start_time = None)
+        self.infector.place_types.append(PlaceType.PrimarySchool)
```

Comment:
What does this has to do with quarantine? Is this to make sure that the effect of place closure will not be applied to people that are in quarantine? If so than we should also check if they are not in isolation.

Reply:
Sorry, I made it the wrong place. It should under place_closure. Thanks!

Reply:
Quarantine and isolation does not remove places from a person place_list, it only scales the foi. This means that if the place is also closed the foi is scaled twice. Does this make sense? Or should we check in place closure of people are not in quarantine/isolation? On the other hand if they are in quarantine/isolation there is a small chance that they still go and infect people and this chance should decrease further when place is closed as well.